build-config/arch/armv8a/crosstool.armv8a.config

```diff
@@ -0,0 +1,559 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Crosstool-NG Configuration
+#
```

Comment:
Same comment about the kernel configuration change below.  We already have a location to store that type of configuration:

```
build-config/kernel/<kernel-version>/linux.<cpu-arch>.config
```